web/src/routes/System.jsx

```diff
@@ -219,20 +221,27 @@ export default function System() {
                         <Tr>
                           <Th>Process</Th>
                           <Th>P-ID</Th>
-                          <Th>fps</Th>
-                          <Th>Cpu %</Th>
+                          <Th>FPS</Th>
+                          <Th>CPU %</Th>
```

Comment:
If you're going to capitalize CPU then you should also capitalize GPU

Reply:
Ah yes! I don't have a GPU in my current test system so can't see that...will ensure I retest on another box in future.